WeakAuras/Init.lua

```diff
@@ -1,20 +1,30 @@
 WeakAuras = {}
 WeakAuras.L = {}
 
-local versionString = GetAddOnMetadata("WeakAuras", "Version");
+local versionStringFromToc = GetAddOnMetadata("WeakAuras", "Version");
+local versionString = "@project-version@"
 --@debug@
 if versionString == "@project-version@" then
   versionString = "Dev"
 end
 --@end-debug@
-WeakAuras.versionString = versionString
+
+WeakAuras.versionString = versionStringFromToc
 WeakAuras.printPrefix = "|cff9900ffWeakAuras:|r "
 WeakAuras.newFeatureString = "|TInterface\\OptionsFrame\\UI-OptionsFrame-NewFeatureIcon:0|t"
 
 WeakAuras.prettyPrint = function(msg)
   print(WeakAuras.printPrefix .. msg)
 end
 
+WeakAuras.versionMismatchPrint = function()
+  WeakAuras.prettyPrint("You need to restart your game client to complete the WeakAuras update!")
+end
+
+if versionString ~= versionStringFromToc and versionString ~= "Dev" then
```

Comment:
This will cause a lot of false positives whenever there is an update which doesn’t add any new files.

Reply:
Yeah well, doesn't hurt either?

Reply:
Well, it’s probably fine. People have awful hygiene when it comes to updating addons anyways.